src/wasm-delegations-fields.def

```diff
@@ -596,6 +596,14 @@ switch (DELEGATE_ID) {
     DELEGATE_END(Try);
     break;
   }
+  case Expression::Id::TryTableId: {
+    DELEGATE_START(TryTable);
+    DELEGATE_FIELD_SCOPE_NAME_USE_VECTOR(TryTable, catchDests);
+    DELEGATE_FIELD_NAME_KIND_VECTOR(TryTable, catchTags, ModuleItemKind::Tag);
+    DELEGATE_FIELD_CHILD(TryTable, body);
+    DELEGATE_END(TryTable);
```

Comment:
I think we also need `catchRefs` here, which probably means we need to add a new `DELEGATE_FIELD_INT_VECTOR`. Without that, hashing etc. will not work well.

I think we need `sentTypes` as well, even though they are just cached metadata. Types can be updated in various passes, and we need to know where to find them (when we update a type, we replace it as they are immutable).

Reply:
Will do after #6182 lands. Adding `DELEGATE_FIELD_INT_VECTOR` and `DELEGATE_FIELD_TYPE_VECTOR` for those required me to add empty definitions for them in `branch-utils.h` and some other places, which I think is because we didn't define `DELEGATE_GET_FIELD` in some places. I did some cleanup related to that in #6182.

Reply:
Done